docs/upgrade.md

```diff
@@ -88,6 +88,13 @@ process, for example:
     dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
     ```
 
+Generally Synapse database schemas are compatiblea cross the following versions.
+The left column is the latest Synapse version which can be rolled back to the right
+column. E.g. Synapse versions v1.13.0 through v1.44.0 can be rolled back safely to
+v1.13.0.
```

Comment:
I'm unsure if this description is clear or if the left-column should maybe be a range of versions?

Reply:
I would say a range, but it doesn't have to be perfect right now